tests/e2e/taqueria-bootstrap.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { spawnSync } from 'child_process';
+import { readdir, readFile, rm, writeFile } from 'fs/promises';
+import { join } from 'path';
+import { generateTestProject } from './utils/utils';
+
+const taqueriaProjectPath = 'e2e/taq-bootstrap';
+
+const expectedConfig = `
```

Comment:
Since these are just strings you can always put them into a data file and then import them for this test just to cut down on what is in this file

Reply:
Sounds like this is a preference, which I'm perfectly fine adopting. Moved to e2e/data/bootstrap-contents.ts.